docs/2-Examples/2-projects_using_tether.md

```diff
@@ -24,17 +24,21 @@ in making something which pops up next to something the user clicks or hovers on
 
 If you're building something that fits that pattern, Drop can make things a little easier.
 
-### [React Datepicker](https://github.com/Hacker0x01/react-datepicker)
+### [Blueprint](http://blueprintjs.com/)
 
-A simple and reusable datepicker component for React
+A React UI toolkit for the web.
+
+### [Bootstrap 4](http://getbootstrap.com/)
+
+Tooltip and popovers from Bootstrap 4 are positioned using Tether
 
-### [ember-tether](https://github.com/yapplabs/ember-tether)
+### [Ember Tether](https://github.com/yapplabs/ember-tether)
 
 An Ember.js-friendly interface for tether.
 
-### [Bootstrap 4](http://getbootstrap.com/)
```

Comment:
I didn't! I just alphabetized it because it seemed like this list was a little bit disorganized